docs/HowTo/Send-Transactions/Revert-Reason.md

```diff
@@ -5,26 +5,34 @@ description: Including revert reason in transactions with Hyperledger Besu
 # Revert reason
 
 In smart contracts, the
-[`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) operation
+[`revert`](https://docs.soliditylang.org/en/v0.8.12/control-structures.html#revert) operation
 triggers an exception to flag an error and revert the current call. The EVM passes back to the
 client an optional string message containing information about the error.
 
 !!! example
 
     ```sol
-    pragma solidity >=0.5.0 <0.7.0;
+    pragma solidity ^0.8.4;
 
     contract VendingMachine {
+        address owner;
```

Comment:
maybe `owner` should be set in a constructor otherwise this example will always have an undefined `owner`.
See for instance https://github.com/ethereum/solidity/blob/develop/docs/contracts/creating-contracts.rst